migrations/1547039077505_bids-create.js

```diff
@@ -0,0 +1,39 @@
+import { Bid } from '../src/Listing'
+import { LISTING_ASSET_TYPES } from '../shared/listing'
+
+const tableName = Bid.tableName
+
+exports.up = pgm => {
+  pgm.createTable(tableName, {
+    id: { type: 'TEXT', primaryKey: true, notNull: true },
+    token_address: { type: 'TEXT', notNull: true },
+    token_id: { type: 'TEXT', notNull: true },
+    bidder: { type: 'TEXT', notNull: true },
+    seller: 'TEXT',
+    price: { type: 'FLOAT', notNull: true },
+    expires_at: { type: 'BIGINT', notNull: true },
+    fingerprint: { type: 'TEXT', notNull: true },
```

Comment:
Because if an Estate is modified I can check if the original fingerprint was changed and temporarily invalidate the bid with a `fingerprintChanged` status. 

If the user changed the Estate again and the fingerprints match, I can set the status as `open` again.

This is useful to not return bids which can be accepted

Reply:
Gotcha